alacritty/src/event.rs

```diff
@@ -1474,6 +1474,13 @@ impl Processor {
                             WinitEvent::RedrawEventsCleared,
                         );
                     }
+
+                    // Update the scheduler after we've done everything, so
+                    // the deadline will be more correct.
+                    *control_flow = match scheduler.update() {
+                        Some(instant) => ControlFlow::WaitUntil(instant),
+                        None => ControlFlow::Wait,
+                    };
```

Comment:
This comment is useful for my review, but "more correct" as a relative term will be completely useless when you don't have the comparison to the old version anymore.

I'd suggest rewording to something like this:

```suggestion
                    // Update the scheduler after event processing, to ensure the event loop deadline is as accurate as possible.
                    *control_flow = match scheduler.update() {
                        Some(instant) => ControlFlow::WaitUntil(instant),
                        None => ControlFlow::Wait,
                    };
```